lib/services/datasources/authzv2.permissions.js

```diff
@@ -14,22 +14,18 @@ async function getPermissions(jwtToken) {
     }
     const querysparams = { applicationId: sources.authzv2.applicationId, jwtToken };
     const headers = { apiKey: sources.authzv2.apiKey };
-    const body = await request.get({
-      url: `${sources.authzv2.url}/permissions`,
-      qs: querysparams,
+    const { data } = await axios.get(`${sources.authzv2.url}/permissions`, {
+      params: querysparams,
       headers,
-      json: true,
     });
-    return body.permissions;
+    return data.permissions;
   } catch (e) {
-    if (e.name === 'StatusCodeError') {
-      const message = (e.error && e.error.title) ? e.error.title : e.message;
+    if (e.status && e.status !== 200) {
```

Comment:
`e.status !== 200`

Status 200 zal niet in catch terechtkomen lijkt mij? En zit de `status` ook op `e`? Volgens de documentatie is dit `e.response.status`.

https://github.com/axios/axios#handling-errors

```suggestion
    if (e.response) {
```

Reply:
Goed gezien. Ik heb het aangepast in de code & testen